### PR TITLE
:bulb: Improved comment accuracy in random_init_board

### PR DESCRIPTION
**Pull request by ickule**

### CODE REVIEW

File: include/random_init_board.h [hunk 1]
```diff
@@ -5,6 +5,6 @@
 #include <time.h>
 
 void random_init_board(int *board, int rows, int cols);
-/* Initialise the board with all cells at 0 */
+/* Initialise the board with a random dead or alive sate */
```
Comment (kaoliiny) [inline include/random_init_board.h:8]: Sate?

File: src/random_init_board.c [hunk 2]
```diff
@@ -1,12 +1,13 @@
 #include "random_init_board.h"
 
 void random_init_board(int *board, int rows, int cols)
-/* Initialise the board with random alive/dead */
+/* Initialise the board with a random dead or alive sate */
```
Comment (kaoliiny) [inline src/random_init_board.c:4]: What is sate?

File: include/random_init_board.h (hunk 1 above)
Comment (kaoliiny) [reply include/random_init_board.h:8]: Pls keep description above the func

Comment (ickule) [reply include/random_init_board.h:8]: The comment part will be handled here along with other functions.
https://github.com/ickule/yagolic/issues/29

File: src/random_init_board.c (hunk 2 above)
Comment (ickule) [reply src/random_init_board.c:4]: Its "state" without a t ^^'

**Review by kaoliiny — APPROVED** [review]

**Pull request merged**